powershell/MSF.Powershell/Runner.cs

```diff
@@ -24,11 +24,11 @@ static Runner()
             _runners = new Dictionary<string, Runner>();
         }
 
-        internal static void Channelise(string id, Int64 channelWriter, Int64 channel, Int64 context)
+        internal static void Channelise(string id, Int64 channelWriter, Int64 context)
         {
             var runner = Get(id);
-            System.Diagnostics.Debug.Write(string.Format("[PSH RUNNER] Channelising {0} with 0x{1:X} - 0x{2:X}", id, channelWriter, context));
-            runner._host.UserInterface.Channelise(channelWriter, channel, context);
+            System.Diagnostics.Debug.Write(string.Format("[PSH RUNNER] Channelising {0} with CW 0x{1:X} - CTX 0x{2:X}", id, channelWriter, context));

```

Comment:
An unrelated observation for this PR, should System.Diagnostics.Debug.Write become a local method instead, that we can switch off globally?